arrows/tests/test_resection_camera.h

```diff
@@ -0,0 +1,176 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <test_eigen.h>
+#include <test_scene.h>
+
+#include <arrows/mvg/projected_track_set.h>
+
+#include <vital/math_constants.h>
+
+using namespace kwiver::vital;
+using namespace kwiver::arrows::mvg;
+using namespace std;
+
+// ----------------------------------------------------------------------------
+// Test camera resection with ideal points.
+TEST ( resection_camera, ideal_points )
+{
+  // Landmarks at random locations
+  landmark_map_sptr landmarks = kwiver::testing::init_landmarks( 128 );
+  landmarks = kwiver::testing::noisy_landmarks( landmarks, 1.0 );
+
+  // camera sequence (elliptical path)
+  camera_map_sptr cameras = kwiver::testing::camera_seq();
+
+  // tracks from projections
+  track_set_sptr tracks = projected_tracks( landmarks, cameras );
+
+  constexpr frame_id_t test_frame = 0;
+
+  // corresponding image points
```

Comment:
I'm guessing a lot of the code from this point on is heavily redundant; would it be possible to refactor this into a helper function? (You'd need to pass in the tolerances and test frame, and it looks like you'd also need to either get back the inliers, or pass a `bool` whether to assert that all points are inliers.)

Reply:
perhaps some helpers  would be helpful, but later, when we actually settle on the API, as it appears to be changing